### PR TITLE
Update jaeger std trf

### PR DESCRIPTION
**Pull request by tiopramayudi**
**What this PR does / why we need it**:

Current implementation of jaeger tracing in UPI standard transformer is broken, several issues on existing implementation
* Standard transformer can't propagate the parent context from the upstream service
* UPI ST not propagate the metadata to the predictor

Below is the result screenshot:

Screen Shot 2023-07-03 at 11 03 20

**Which issue(s) this PR 

fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

- [ ] Added unit test, integration, and/or e2e tests
- [x] Tested locally
- [ ] Updated documentation
- [ ] Update Swagger spec if the PR introduce API changes
- [ ] Regenerated Golang and Python client if the PR introduce API changes

### CODE REVIEW

File: api/cmd/transformer/main.go [hunk 1]
```diff
@@ -60,6 +63,18 @@ type AppConfig struct {
...
+
+	Tracing JaegerTracing
+}
+
+// JaegerConfig holds configuration about jaeger tracing
+type JaegerTracing struct {
+	// Disabled the tracing, will create NoOpTracerProvider
+	Disabled bool `envconfig:"JAEGER_DISABLED" default:"true"`
+	// Collector endpoint URL
+	CollectorURL string `envconfig:"JAEGER_ENDPOINT"`
```
Comment (pradithya) [inline api/cmd/transformer/main.go:75]: Can we explicitly use `JAEGER_COLLECTOR_URL` instead of calling it `JAEGER_ENDPOINT` ?

Comment (pradithya) [reply api/cmd/transformer/main.go:75]: Can you remove unused JAEGER constants under constant.go?

File: api/pkg/transformer/feast/validator_test.go [hunk 2]
```diff
@@ -753,7 +753,7 @@ func TestValidateTransformerConfig(t *testing.T) {
 					},
 				},
 			},
-			wantError: errors.New("udf compilation failed: unknown func unknown (1:1)\n | unknown()\n | ^"),
+			wantError: errors.New("udf compilation failed: unknown name unknown (1:1)\n | unknown()\n | ^"),
```
Comment (pradithya) [inline api/pkg/transformer/feast/validator_test.go:756]: Is this intended change?

**Review by pradithya — APPROVED** [review]
LGTM, a few nitpicks below. Can you share the screenshots of the standard transformer trace in the PR description?

File: api/pkg/transformer/feast/validator_test.go (hunk 2 above)
Comment (tiopramayudi) [reply api/pkg/transformer/feast/validator_test.go:756]: yes, this is due to update on `expr` library

File: api/cmd/transformer/main.go (hunk 1 above)
Comment (tiopramayudi) [reply api/cmd/transformer/main.go]: updated

**Comment by pradithya** [comment]
@tiopramayudi can it be merged?

**Comment by tiopramayudi** [comment]
> @tiopramayudi can it be merged?

yes, I'm merging it now

**Pull request merged**